src/recovery.rs

```diff
@@ -391,11 +403,19 @@ impl Recovery {
                 self.smoothed_rtt = Some(latest_rtt);
 
                 self.rttvar = latest_rtt / 2;
+
+                self.minmax_filter.reset(update_time, latest_rtt);
             },
 
             Some(srtt) => {
                 self.min_rtt = cmp::min(self.min_rtt, latest_rtt);
 
+                self.windowed_min_rtt = self.minmax_filter.running_min(
```

Comment:
We can simply use this filter to update self.min_rtt. No need to keep separate value.